### DISCUSSION TITLE
Compatibility of MACE Potential with ML-IAP Package in Standard LAMMPS

### DISCUSSION THREAD

**Discussion by scottedmuller-pnnl** [ACEsuit/mace#583] [Q&A] [answered]
Hello,

I have been working with a MACE-derived interatomic potential (`*.model`, converted to a `*.pt` file by `mace_create_lammps_model`) and would like to know if it is possible to use this potential with the `mliap` pair_style from the standard LAMMPS distribution (not the ACEsuit version). Specifically:
1. Can a MACE potential file (e.g., `my_mace.model-lammps.pt`) be used with the ML-IAP package (`pair_style mliap`) in standard LAMMPS?
2. If so, how can a descriptor file (e.g., a `*.yace` file for the ace descriptor style) be generated to accompany the `*.pt` file?
3. If it is not currently possible to use a MACE potential with standard LAMMPS, are there any plans to implement this in the future to make MACE potentials more transferrable across different LAMMPS builds?
4. Lastly, are there any fundamental components of a MACE potential that may be inherently incompatible with the `mliap` pair_style in standard LAMMPS?

I appreciate any guidance or suggestions you can offer. Thanks!

**Comment by wcwitt** [comment, marked as answer]
The short answer is no, you need to use our fork. This may change with time.
